geoshape/operations/default.json

```diff
@@ -0,0 +1,62 @@
+    {
+      "name": "index-append-linestrings",
+      "operation-type": "bulk",
+      "bulk-size": {{bulk_size | default(100)}},
```

Comment:
As I've mentioned in my feedback for the README, I'm not sure whether we should have a different bulk size per corpus?